### ISSUE TITLE
exception exit: noproc on OTP 21.0-rc1

### ISSUE THREAD

**Issue by leoliu** [klacke/yaws#334]
I can't wait to use OTP 21.0 which is looking delicious bringing in a lot of long-awaited features. I tried Yaws (built on 20.3.2) with 21.0-rc1 with no luck. Run `yaws:start_embedded(Dir)` to immediately get a crash dump. Looks like yaws_log is having a hard time on OTP 21.0.

```
** When Server state == {state,undefined,[],true}
** Reason for termination == 
** {{noproc,
        {gen_server,call,
            [yaws_log,
             {setup,
                 {gconf,
                     "/Users/leo/sources/testapp/_build/default/lib/yaws",
                     false,612,".",
                     ["/Users/leo/sources/testapp/_build/default/lib/yaws/examples/ebin"],
                     [],[],30000,nolimit,400,1000000,8000,nolimit,[],10240,[],
                     10000000,30,
                     ["/Users/leo/sources/testapp/_build/default/lib/yaws/examples/include"],
                     "/usr/bin/php-cgi","Yaws 2.0.5","default",false,[],8,
                     undefined,
                     [inet],
                     yaws_session_server,undefined,120000,3600000,disable},
                 []},
             infinity]}},
    [{gen_server,call,3,[{file,"gen_server.erl"},{line,223}]},
     {yaws_server,init2,5,[{file,"yaws_server.erl"},{line,243}]},
     {yaws_server,handle_call,3,[{file,"yaws_server.erl"},{line,305}]},
     {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,661}]},
     {gen_server,handle_msg,6,[{file,"gen_server.erl"},{line,690}]},
     {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
```

**Comment by vinoski** [comment]
Currently working on this with rc2.

**Comment by vinoski** [comment]
Fixed at commit ba31d14.
